src/frameworks.js

```diff
@@ -0,0 +1,40 @@
+/* Copyright 2015 Mozilla Foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/* globals PDFJS, require, module */
+
+// included from api.js for GENERIC build
+
+'use strict';
+
+var useRequireEnsure = false;
+if (typeof module !== 'undefined' && module.require) {
```

Comment:
Hm... maybe it's better to use `System.import` polyfill instead of `require.ensure`?
https://github.com/ModuleLoader/es6-module-loader
I believe, pdf.js is used more on browser side, so using ES6 modules seems more logical.